inspirehep/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -31,9 +31,11 @@
 {% macro render_record_authors(is_brief, number_of_displayed_authors=10, show_affiliations=true) %}
   {% if record.collaboration %}
     {% set collaboration_displayed = [] %}
-    {% for collaboration in record.collaboration if collaboration != None %}
-      <a href="/search?p=collaboration:'{{ collaboration }}'">{{ collaboration }}</a>
+    {% for collaboration in record.collaboration %}
+      {% if collaboration['value'] %}
```

Comment:
Cool. I have seen keyerrors before in other cases but I cannot recall which ones. Yeah in jinja templates it is often more appropriate to use `foo.bar` or `foo['bar']` instead of `foo.get('bar')`